docs/application/tizen-studio-0/setup/about_ide.md

```diff
@@ -0,0 +1,35 @@
+# About Tizen Studio
```

Comment:
Title have to be "Tizen Studio". 
Overview for the title of menu tree is better.

Reply:
I have made the necessary changes, please review and give your valuable comments.